LDAR_Sim/src/programs/method.py

```diff
@@ -219,16 +219,10 @@ def deploy_crews(self, workplan: Workplan, weather, daylight) -> CrewDeploymentS
         for crew in self._crew_reports:
             # TODO : if method is daylight sensitive, check for max daylight
             crew.day_time_remaining = day_time_remaining
+            crew.deployed = False
             priority_queue.put((-crew.day_time_remaining, crew.crew_id, crew))
 
-        # If the cost type for the method is per day, calculate the deployment cost for day
-        # based off the number of crews being deployed
-        if self.cost_type == self.PER_DAY_COST:
-            if self._deployment_type == pdc.Deployment_Types.STATIONARY:
```

Comment:
Have we tested this with stationary? The case for stationary still holds true, but if new functionality work too I see no issue

Reply:
Updated for stationary, as stationary by default only has one crew.